package.json

```
@@ -22,11 +22,13 @@
     "lib"
   ],
   "scripts": {
-    "pretest": "XMLDOM_ASSERT_STRICT=1 npm run test:assert",
-    "test": "vows 'test/**/*.vows.js'",
-    "test:assert": "vows test/assert.vows.js",
     "start": "nodemon --watch package.json --watch lib --watch test --exec 'npm --silent run test'",
-    "stryker": "stryker run"
+    "stryker": "stryker run",
+    "test": "npm-run-all test:assert:strict test:assert test:unit",
```

I think we can do this:

```suggestion
    "test": "npm-run-all test:assert:strict test:unit",
```